src/backend/app/projects/project_routes.py

```diff
@@ -657,3 +658,54 @@ async def regulator_approval(
             status_code=HTTPStatus.BAD_REQUEST,
             detail=f"An error occurred: {str(e)}",
         )
+
+
+@router.get("/{project_id}/waypoints", tags=["Projects"])
+async def get_project_waypoints_geojson(
+    project_id: uuid.UUID,
+    side_overlap: float,
+    front_overlap: float,
+    altitude_from_ground: float,
+    gsd_cm_px: float,
+    meters: float = 100,
+    is_terrain_follow: bool = False,
```

Comment:
can we get all these informations like overlaps and all from the projects table. I think we already store them. So, asking again in the api does not make sense.